pkg/sdk/testint/tags_integration_test.go

```diff
@@ -0,0 +1,271 @@
+package testint
+
+import (
+	"context"
+	"errors"
+	"testing"
+
+	"github.com/Snowflake-Labs/terraform-provider-snowflake/pkg/sdk"
+	"github.com/Snowflake-Labs/terraform-provider-snowflake/pkg/sdk/internal/collections"
+	"github.com/Snowflake-Labs/terraform-provider-snowflake/pkg/sdk/internal/random"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestInt_Tags(t *testing.T) {
+	client := testClient(t)
+	ctx := context.Background()
+
+	databaseTest, databaseCleanup := createDatabase(t, client)
```

Comment:
there is already PR merged with predefined db, you can adjust it here or in the next PR, up to you